soc/arm/ti_k3/j721e/linker.ld

```diff
@@ -0,0 +1,20 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/arm/aarch32/cortex_a_r/scripts/linker.ld>
+
+SECTIONS
+{
+    SECTION_PROLOGUE(.resource_table,,)
+    {
+        KEEP(*(.resource_table*))
+    } > RAM
```

Comment:
Why is this RPMSg stuff is needed?

Reply:
J721e SoC is a heterogeneous has A72 (Linux), cortex R5 (Zephyr). The zephyr image is loaded to cortex r5 through remoteproc from A72 running Linux, so remoteproc requires .resource_table section in the zephyr.elf file.